tests/integration/clientv3/naming/resolver_test.go

```diff
@@ -15,56 +15,103 @@
 package naming_test
 
 import (
+	"bytes"
 	"context"
 	"testing"
+	"time"
 
+	etcdnaming "go.etcd.io/etcd/client/v3/naming"
 	"go.etcd.io/etcd/client/v3/naming/endpoints"
-	"go.etcd.io/etcd/client/v3/naming/resolver"
-	"google.golang.org/grpc"
-
 	"go.etcd.io/etcd/pkg/v3/testutil"
 	"go.etcd.io/etcd/tests/v3/integration"
+
+	"google.golang.org/grpc"
+	testpb "google.golang.org/grpc/test/grpc_testing"
 )
 
 // This test mimics scenario described in grpc_naming.md doc.
 
 func TestEtcdGrpcResolver(t *testing.T) {
-	t.Skip("Not implemented yet")
-
 	defer testutil.AfterTest(t)
 
-	// s1 :=  // TODO: Dummy GRPC service listening on 127.0.0.1:20000
-	// s2 :=  // TODO: Dummy GRPC service listening on 127.0.0.1:20001
+	s1PayloadBody := []byte{'1'}
+	s1 := &integration.StubServer{
```

Comment:
Probably worth to write helper method for s1/s2 creation as the logic is shared.

Reply:
Done